linux/system/repo.sls

```diff
@@ -132,6 +132,9 @@ linux_repo_{{ name }}:
   {%- if repo.key_server is defined %}
   - keyserver: {{ repo.key_server }}
   {%- endif %}
+  {%- if repo.key_url is defined and repo.key_url.startswith('salt://') %}
```

Comment:
we should support even http:// and https:// anyway the condition to startswith is not much logical, on the other hand we should get rid of the curl usage above

Reply:
http/https are supported already with the curl command I think

Reply:
Will be fixed properly with this: https://github.com/salt-formulas/salt-formula-linux/pull/155